### PR TITLE
FE-402: Adventures in Web Animation

### PR DESCRIPTION
**Pull request by nicoRDL**
## Description
- adds css animations, transitions, and transforms to different elements making website more dynamic and interactable

## Related Ticket
https://codeandtheory.atlassian.net/browse/FE-655

## Testing Instructions
1. Pull branch FE-402/FE-655
2. Evaluate the animations and css updates

## Risks
N/A

### CODE REVIEW

File: scss/components/_typography.scss [hunk 1]
```diff
@@ -303,26 +303,91 @@
...
+    transition: text-shadow 0.5s;
+
+    &:hover {
+      text-shadow: 0.125rem 0.25rem 0.5rem $color-nav-shadow-1;
+    }
+  }
+}
+
+.author-card {
+  &-content {
```
Comment (lycoangelo) [inline scss/components/_typography.scss]: This could be rewritten as `.author-card-content` instead.

File: scss/components/_typography.scss [hunk 2]
```diff
@@ -303,26 +303,91 @@
...
+    opacity: 0;
+    &__name {
```
Comment (lycoangelo) [inline scss/components/_typography.scss]: Please add a new line before every selector.

File: scss/layout/_containers.scss [hunk 3]
```diff
@@ -547,3 +552,51 @@
...
+
+    &-content {
+      transition: opacity 0.8s 0s;
+    }
+
+    &:hover > &-content {
+      opacity: 1;
+    }
+  }
+}
```
Comment (lycoangelo) [inline scss/layout/_containers.scss]: Please convert this scope to a BEM pattern.

**Review by lycoangelo — CHANGES_REQUESTED** [review]

File: scss/components/_typography.scss (hunk 1 above)
Comment (nicoRDL) [reply scss/components/_typography.scss]: Noted. Updated code here e9a8403655a942a7d2a1ed379419d8edc68802b7

File: scss/layout/_containers.scss (hunk 3 above)
Comment (nicoRDL) [reply scss/layout/_containers.scss]: Code updated, also updated typography.scss file to match BEM pattern 
e9a8403655a942a7d2a1ed379419d8edc68802b7

**Pull request merged**